tests/test_add_meta_tag.py

```diff
@@ -10,7 +10,9 @@ class TestAddMetaTag(unittest.TestCase):
 
     def setUp(self):
         if not os.path.exists("/opt/snare/pages/test"):
+            self.path_exists = False
```

Comment:
what is the purpose of this flag variable? if you have this directory, you will override index file, it might be crucial. And also it will be always True

I guess it's the problem that we can change default folder '/opt/snare/pages'. In this case, probably will be better to create folder with random character sequence and not just use test

and makedirs by default raises the exception, if file exists

Reply:
Looking at the other prs I understand the purposes of this flag and it can be used as a temporal solution before we change our structure :)